lib/ws_servers/api/open_auth_bitfinex_connection.js

```diff
@@ -53,24 +68,51 @@ module.exports = ({ ws, apiKey, apiSecret, authToken, dms, d, wsURL, restURL, is
   })
 
   client.on('ws2:open', () => {
-    notifyInfo(ws, 'Connected to Bitfinex', ['connectedTo', { target: 'Bitfinex' }])
+    notifyInfo(ws, 'Connected to Bitfinex', [
+      'connectedTo',
+      { target: 'Bitfinex' }
+    ])
   })
 
   client.on('ws2:event:auth:success', async () => {
-    notifySuccess(ws, 'Authenticated with Bitfinex', ['authenticatedWith', { target: 'Bitfinex' }])
+    notifySuccess(ws, 'Authenticated with Bitfinex', [
+      'authenticatedWith',
+      { target: 'Bitfinex' }
+    ])
     send(ws, ['data.client', 'bitfinex', mode, WS_CONNECTION.OPENED])
     const userInfo = await client.getUserInfo()
     if (userInfo) {
-      send(ws, ['info.username', isPaper ? 'paper' : 'main', userInfo.username])
+      send(ws, [
+        'info.username',
+        isPaper ? 'paper' : 'main',
+        userInfo.username
+      ])
     }
+
+    // await sendRecurringAlgoOrderList(
```

Comment:
why is this commented?

Reply:
Hugo decided to remove Recurring feature from upcoming release. When we back to recurring, this could be uncommented